templates/bundles/PrestaShopBundle/Admin/Common/Grid/Columns/Content/image.html.twig

```diff
@@ -22,5 +22,8 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
+ 
+{% set width  = record[column.options.width] %}
+{% set height = record[column.options.height] %}
 
-<img src="{{ record[column.options.src_field] }}">
+<img src="{{ record[column.options.src_field] }}" {% if width %} style="width: {{ width }}; height: {{ height }};" {% endif %}>
```

Comment:
if width doesn't mean the height is defined? :thinking:

Reply:
@PierreRambaud  `height` is `auto` by default. See defaults definitions please.

Reply:
That doesn't mean you can't put empty value :)

Reply:
```suggestion
<img src="{{ record[column.options.src_field] }}" style="{% if width %} width: {{ width }}; {% endif %} height: {{ height }};">
```

Reply:
```suggestion
<img src="{{ record[column.options.src_field] }}"{% if width is not empty or height is not empty %} style="{% if width is not empty %}width:{{ width }};{% endif %}{% if height is not empty %}height:{{ height }};{% endif %}"{% endif %}>
```